README.md

```diff
@@ -36,7 +36,7 @@ context = RDF::Graph.new << [uri, RDF::Vocab::DC.title, "Some Title"]
 
 program = Ldpath::Program.parse my_program
 output = program.evaluate uri, context: context
-# => { ... }
+# => {"title"=>["Some Title"]}
```

Comment:
Added the expected result should be for documentation purposes.

Reply:
That documents the normal case, but the thing being added in this PR with `maintain_literals` is still (I think?) entirely undocumented. As the simplest possible thing, since ldpath doesn't seem super documented in general anyway but that's not a problem for this PR, what about just adding antoher examples to the README with `maintain_literals: true`, and what that would return?

Also, just confirm you think `maintain_literals` is the right name of this arg? Preferable to, oh, `use_literals` or `return_literals` or `return_rdf_literals` or `rdf_literals: true` or something like that? I think it's up to you, I just think it's good to at least publicly consider if it's the best one we can think of before locking it into a release from which it can't be changed without backwards incompat. 

I'm not totally locked into any of these ideas, just trying my best to give a responsible non-rubberstamping review on code I'm not actually at all familiar with. Disagreement welcome you can just tell me "nah, I know this code better and it's right how it is", and I'll just approve! Thanks!